app/uk/gov/hmrc/gform/gform/processor/FormProcessor.scala

```diff
@@ -392,6 +395,15 @@ class FormProcessor(
     } yield res
   }
 
-  def getSectionTitle4Ga(processData: ProcessData, sectionNumber: SectionNumber): SectionTitle4Ga =
-    sectionTitle4GaFactory(processData.formModel(sectionNumber), sectionNumber)
+  def getSectionTitle4Ga(processData: ProcessData, sectionNumber: SectionNumber)(implicit
```

Comment:
Why not just take
```
(implicit  sse: SmartStringEvaluator)
```
?

Reply:
we can't do that here. It should be a brand new SSE with fresh data from the page not from mongo

Reply:
Ok, I now see why my suggestion would be an issue.